admin-dev/themes/new-theme/js/components/form/choice-tree.js

```diff
@@ -47,15 +48,28 @@ export default class ChoiceTree {
       this._toggleTree($action);
     });
 
+    this.$container.on('click', '.js-unselect-all', (event) => {
+      event.preventDefault();
+
+      const hasCheckedElement = this.$container.find('.js-input-wrapper input:checked').length > 0;
```

Comment:
Is `.js-input-wrapper` class unique enough? What happen if other elements that are not part of `choice-tree` will use it?

Reply:
The only unique selector in this selector component is `js-input-wrapper`.  Since it is `this.$container.find('.js-input-wrapper)` then I think there is no problem that other elements could possibly use it.